#!/bin/sh /usr/share/dpatch/dpatch-run

## 99_clean.dpatch - Tobias Grimm <tg@e-tobi.nez>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fixes clean target

@DPATCH@
--- vdr-plugin-live-0.1.0.orig/pages/Makefile
+++ vdr-plugin-live-0.1.0/pages/Makefile
@@ -53,4 +53,4 @@
 	$(AR) r $@ $^
 
 clean:
-	@rm -f *~ *.o core* libpages.a $(OBJS:%.o=%.cpp)
+	@rm -f *~ *.o core* libpages.a $(OBJS:%.o=%.cpp) $(DEPFILE)
